File: src/main/java/com/lee/guess/Game.java
package com.lee.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {

    public static void main(String[] args) {

        Random random = new Random();
        int secret = random.nextInt(10)+1;
        Scanner scanner = new Scanner(System.in);
        System.out.println(""+secret);
        int number = 0;
        while (number != secret){
            System.out.println("Please input a number :");
            number = scanner.nextInt();
            if (number <secret){
                System.out.println("higher");
            }else if (number>secret){
                System.out.println("lower");
            }else{
                System.out.println("Great, number is "+secret);
            }

        }

    }
}

File: src/main/kotlin/com/lee/Game.kt
package com.lee

import java.util.*

fun main() {

    var secret = Random().nextInt(10)+1
    println(secret)
    val scanner = Scanner(System.`in`)
    var number =0
    println("Please input your number :")
    while (number != secret){
        number = scanner.nextInt()

//        when(secret>0){
//            number>secret -> println("lower")
//            number < secret -> println("higher")
//            else -> println("Great number is $number")
//        }
        if (number>secret){
            println("lower")
        }else if(number < secret){
            println("higher")
        }else{
            println("Great number is $number")
        }
    }


}